test/index.test.js

```diff
@@ -174,13 +171,14 @@ describe('index test', () => {
         });
 
         it('parses a valid yaml', () =>
-            validator(loadData(VALID_FULL_PIPELINE_TEMPLATE_PATH)).then(config => {
+            validator(loadData('valid_full_pipeline_template.yaml')).then(config => {
```

Comment:
are any changes needed in this file?

Reply:
just easier to read, i don't see any reason to keep it as global constant

Reply:
please put this back as a constant, it is not needed to be changed and it should follow the semantic of the rest of tests in general, which is reading from a file that is set as a const var.